File: trunk/server/src/main/java/org/ngame/protocol/battle/BaseLvResponse.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.BaseResponse;
import org.ngame.protocol.annotation.Proto;
import org.ngame.protocol.domain.BaseInfo;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = -22, description = "基地升级")
public class BaseLvResponse extends BaseResponse
{
	@Protobuf(description = "玩家id")
	private String pid;
	@Protobuf(description = "资源数")
	private int resource;
	@Protobuf(description = "base lv")
	private int baseLv;
}

File: trunk/server/net/Protocols/BaseLvResponse.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 基地升级 协议:-22
    /// </summary>
	[Proto(value=-22,description="基地升级")]
	[ProtoContract]
	public class BaseLvResponse
	{
        /// <summary>
        ///  玩家id
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string pid;
        /// <summary>
        ///  资源数
        /// </summary>
		[ProtoMember(4, IsRequired = false)]
		public int resource;
        /// <summary>
        ///  base lv
        /// </summary>
		[ProtoMember(5, IsRequired = false)]
		public int baseLv;
        /// <summary>
        ///  是否成功
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public bool success;
        /// <summary>
        ///  错误消息
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string info;

	}
}